openupgrade_scripts/scripts/account/15.0.1.2/end-migration.py

```diff
@@ -0,0 +1,36 @@
+from openupgradelib import openupgrade
+
+
+def _fast_fill_account_payment_outstanding_account_id(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_payment ap
+        SET outstanding_account_id = apml.payment_account_id
+        FROM account_payment_method_line apml
+        WHERE apml.id = ap.payment_method_line_id
+            AND ap.payment_type IN ('inbound', 'outbound')""",
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE account_payment ap
+        SET outstanding_account_id =
+            CASE
+                WHEN ap.payment_type = 'inbound'
+                THEN c.account_journal_payment_debit_account_id
+                WHEN ap.payment_type = 'outbound'
+                THEN c.account_journal_payment_credit_account_id
```

Comment:
Would it make sense to use the `payment_credit_account_id` and the `payment_debit_account_id` of the `account.payment`'s `journal_id` here? I see them being swapped out in this diff: https://github.com/odoo/odoo/commit/04522f01e6fd#diff-143d17de807d23650088a8c12f0a5b5cc2246b1b51e0bb7634e85247b6e535eaL259-R275

Reply:
> The previous query is relevant here as well. If `account_payment_method_line.payment_account_id` is not populated in this module, what would be the logical place to do so? And this also ties in with [creating all the payment method lines here](https://github.com/OCA/OpenUpgrade/pull/3505/files#r993456741) rather than in the depending payment modules.

Remember this is an end-migration. When this is executed, the scripts in payment and check_printing should have been executed already. And the scripts in pre-migration that populates `account_payment_method_line` always fill `payment_account_id`.